dataactcore/interfaces/function_bag.py

```diff
@@ -350,4 +350,23 @@ def getErrorMetricsByJobId(job_id, include_file_types=False, severity_id=None):
             record_dict['source_file'] = FILE_TYPE_DICT_ID.get(result.file_type_id, '')
             record_dict['target_file'] = FILE_TYPE_DICT_ID.get(result.target_file_type_id, '')
         result_list.append(record_dict)
-    return result_list
\ No newline at end of file
+    return result_list
+
+def get_cross_file_combos():
+    """Return a list of source and target file types used in cross-file validations."""
```

Comment:
Good point--will work on this to make the terminology more consistent and understandable.